fcfs.c

```diff
@@ -52,102 +52,21 @@ static void findAverageTime(process *ptr)
 }
```

Comment:
looks like you are undoing Sagar's changes on this file. your commit shouldn't modify fcfs.c